### PR TITLE
feat: add NepalLink Network

### PR DESCRIPTION
**Pull request by aayush105**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

**Review by drishyakarki — COMMENTED** [review]
Does this place have a landline number?

**Comment by aayush105** [comment]
> Does this place have a landline number?

Yess, this company has a landline number.

**Review by drishyakarki — APPROVED** [review]
LGTM!!

File: data/companies.json [hunk 1]
```diff
@@ -592,6 +592,22 @@
...
     },
+    {
+      "name": "NepalLink Network",
+      "establishedAt": 1106467200,
+      "location": "Kathmandu, Nepal",
+      "website": "https://www.nepallink.net/",
+      "email": "info@nepallink.net",
+      "phone": "+977-14260822",
+      "socials": {
+      "facebook": "https://www.facebook.com/ncompute/",
```
Comment (susanhsrestha) [inline data/companies.json:603]: To make this section's formatting better, use prettier.

File: data/companies.json [hunk 2]
```diff
@@ -592,6 +592,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "NepalLink Network",
+      "establishedAt": 1106467200,
+      "location": "Kathmandu, Nepal",
```
Comment (susanhsrestha) [inline data/companies.json:598]: Add a few location-specific details. Do some research; there is different location information on Google Maps and the official website.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 2 above)
Comment (aayush105) [reply data/companies.json:598]: so which one should I include then the google location or the website location? And in facebook as welll as in Linkedin the location is Kathmandu, thats why I have added the location Kathmandu, Nepal.
@susanhsrestha

File: data/companies.json (hunk 1 above)
Comment (aayush105) [reply data/companies.json:603]: Sure I will fix this section @susanhsrestha

File: data/companies.json (hunk 2 above)
Comment (susanhsrestha) [reply data/companies.json:598]: Okay, proceed with Kathmandu, Nepal.

**Comment by aayush105** [comment]
@thenishantsapkota and  @TheLearneer  please  review this and merge it.

**Review by susanhsrestha — APPROVED** [review]
LGTM!

**Pull request merged**